### PR TITLE
LLVM 4.0

### PR DESCRIPTION
**Pull request by kripken**
See https://github.com/kripken/emscripten-fastcomp/pull/166

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -269,6 +269,7 @@ a license to everyone to use it as detailed in LICENSE.)
 * Vilibald Wanča <vilibald@wvi.cz>
 * Alex Hixon <alex@alexhixon.com>
 * Vladimir Davidovich <thy.ringo@gmail.com>
+* Dylan McKay <dylanmckay34@gmail.com>
```
Comment (dylanmckay) [inline AUTHORS]: My email address is now `me@dylanmckay.io`

Comment (kripken) [inline AUTHORS]: Oh ok, do you want to submit a PR to this branch, with that change?

Comment (dylanmckay) [inline AUTHORS]: Good idea, done in #4937

**Comment by kripken** [comment]
Merged in 1d81afe

**Pull request closed without merging**